admin/components/Post/Editors/plugins/blockquote/BlockquoteKeyboardShortcut.js

```diff
@@ -0,0 +1,5 @@
+const BlockquoteKeyboardShortcut = () => {
+    return;
+};
+
+export default BlockquoteKeyboardShortcut;
```

Comment:
This function seems very weird and is used with some params in `Post/Editors/plugins/blockquote/index.js` but doesn't return anything